Assignment2/GNajeral.ttl

```diff
@@ -0,0 +1,18 @@
+
+@prefix c: <http://www.tarea2/class01>.
+@prefix s: <http://www.tarea2/sensor029>.
+@prefix pc: <http://www.tarea2/computer101>.
+@prefix m: <http://www.tarea2/measurement8401>.
+@prefix u: <http://www.tarea2/user10A>.
```

Comment:
It would be more common to have a single prefix of the schema that is being used, and then use it throughout the example

Reply:
Fixed